app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/injection/module/OutputActivityModule.java

```diff
@@ -1,7 +1,16 @@
 package com.benoitquenaudon.daggerbundleinjectiondemo.injection.module;
 
+import com.benoitquenaudon.daggerbundleinjectiondemo.BundleService;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.SomeInput;
+import com.benoitquenaudon.daggerbundleinjectiondemo.injection.scope.ScopeActivity;
 import com.benoitquenaudon.daggerbundleinjectiondemo.ui.output.OutputActivity;
 import dagger.Module;
+import dagger.Provides;
 
-@Module abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+@Module public abstract class OutputActivityModule implements BaseAppCompatActivityModule<OutputActivity> {
+  public static final String EXTRA_SOME_INPUT = "com.benoitquenaudon.demo.SOME_INPUT";
```

Comment:
Wouldn't be nice to have those constants in one place (as the former `BundleModule`) to avoir duplicates/conflicts?